Taskfile.yml

```diff
@@ -283,3 +283,38 @@ tasks:
         sh: cat internal/util/version/commit.txt
       VERSION:
         sh: cat internal/util/version/version.txt
+
+  build:
+    deps: [gen-version]
+    cmds:
+      - go build -v -o ./bin/ ./cmd/ferretdb
+
+  nfpm_deb:
+    deps: [build]
+    vars:
+      VERSION:
+
+    dir: build/deb
+    cmds:
+      - echo ${VERSION}
+      - ../../bin/nfpm pkg --config ../../.nfpm.yml --packager deb --target ferretdb_${VERSION}.deb
+    env:
+      VERSION:
+        sh: cat ../../internal/util/version/version.txt | grep -oP '^(.+?)(?=-)'
+
+  nfpm_rpm:
+    deps: [build]
+    vars:
+      VERSION:
+        sh: cat ../../internal/util/version/version.txt
+    dir: build/rpm
+    cmds:
+      - ../../bin/nfpm pkg --config ../../.nfpm.yml --packager rpm --target ferretdb_${VERSION}.rpm
+    env:
+      VERSION:
+        sh: cat ../../internal/util/version/version.txt | grep -oP '^(.+?)(?=-)'
```

Comment:
```
task: Failed to run task "packager": task: Command "cat ../../internal/util/version/version.txt | grep -oP '^(.+?)(?=-)'" failed: exit status 2
```

Let's wait until #798 is merged, then change generate.go to generate another file with a clean version